utils/api/server.ts

```diff
@@ -19,7 +19,11 @@ import { contract } from './contract';
 
 const s = initServer();
 
-export const server = s.router(contract, {
+// FIXME: Add these routes back in
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const { customScript, community, ...contractWithScriptsAndCommunity } = contract;
+
+export const server = s.router(contractWithScriptsAndCommunity, {
```

Comment:
Note: I did not implement any server routes for the scripts or contract, that would warrant a lot more testing. As such, these routes do not have validation.

I will convert these routes to ts-rest routes when I do #2850